app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -48,6 +48,8 @@ sealed class RouteAction(
     ) : RouteAction(TelemetryEventMethod.show, TelemetryEventObject.dialog) {
         class FingerprintDialog(@StringRes title: Int, @StringRes subtitle: Int? = null) :
             DialogFragment(dialogTitle = title, dialogSubtitle = subtitle)
+        class OnboardingSecurityDialog(@StringRes title: Int, @StringRes subtitle: Int? = null) :
```

Comment:
I would use the `RouteAction.Dialog` class here, not the `DialogFragment` -- a whole fragment for this is overkill.